app/src/models/EmissionsFactor.ts

```diff
@@ -70,6 +74,16 @@ export class EmissionsFactor
   getInventory!: Sequelize.BelongsToGetAssociationMixin<Inventory>;
   setInventory!: Sequelize.BelongsToSetAssociationMixin<Inventory, InventoryId>;
   createInventory!: Sequelize.BelongsToCreateAssociationMixin<Inventory>;
+
+  // EmissionsFactor belongsTo Methodology via methodologyId
+  methodology!: Methodology;
+  getMethodology!: Sequelize.BelongsToGetAssociationMixin<Methodology>;
+  setMethodology!: Sequelize.BelongsToSetAssociationMixin<
+    Methodology,
+    MethodologyId
+  >;
+  createMethodology!: Sequelize.BelongsToCreateAssociationMixin<Methodology>;
```

Comment:
If you have added a new relation here, make sure to also add the correspoding relationship definitions (hasOne/ hasMany/ belongsTo etc.) to `init-models.ts`.
Otherwise you will get runtime errors when trying to join this in a Sequelize query with `includes`.